### PR TITLE
Implement CVAR_LATCH for new cvars

### PR DESCRIPTION
**Pull request by slipher**
To do before merging: Prevent `The change will take effect after restart` messages from being printed when the cvars are set from command line/configs.

### CODE REVIEW

**Comment by illwieckz** [comment]
thanks for doing this!

**Comment by illwieckz** [comment]
I rebased #262 over this and it works correctly on my end. LGTM.

File: src/common/Cvar.h [hunk 1]
```diff
@@ -46,6 +46,7 @@ namespace Cvar {
         TEMPORARY  = BIT(8), // The cvar is temporary and is not to be archived (overrides archive flags)
         CHEAT      = BIT(9), // The cvar is a cheat and should stay at its default value on pure servers.
         USER_ARCHIVE = BIT(14), // The cvar is saved to the configuration file at user request
+        LATCH      = BIT(15), // The cvar's value can be changed only after a cgame restart. Intentionally different from CVAR_LATCH.
```
Comment (illwieckz) [inline src/common/Cvar.h]: Just for my knowledge, why it has to be different? maybe a comment explaining why would be good, that's my only concern.

Comment (slipher) [reply src/common/Cvar.h]: Done.

**Comment by illwieckz** [comment]
good.

**Pull request merged**